lib/mrtrix3/dwi2mask/synthstrip.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2008-2023 the MRtrix3 contributors.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# Covered Software is provided under this License on an "as is"
+# basis, without warranty of any kind, either expressed, implied, or
+# statutory, including, without limitation, warranties that the
+# Covered Software is free of defects, merchantable, fit for a
+# particular purpose or non-infringing.
+# See the Mozilla Public License v. 2.0 for more details.
+#
+# For more details, see http://www.mrtrix.org/.
+
+import shutil
+import argparse
+import os
+import warnings
+from mrtrix3 import MRtrixError
+from mrtrix3 import app, run
+
+
+DEFAULT_CLEAN_SCALE = 2
+SYNTHSTRIP='mri_synthstrip'
+
+
+def usage(base_parser, subparsers): #pylint: disable=unused-variable
+  parser = subparsers.add_parser('synthstrip', parents=[base_parser])
+  parser.set_author('Ruobing Chen(chrc@student.unimelb.edu.au)')
+  parser.set_synopsis('Use the Synthstrip heuristic (based on skull-stripping method)')
+  parser.add_argument('input',  help='The input DWI series')
+  parser.add_argument('output', help='The output mask image')
+  options=parser.add_argument_group('Options specific to the \' Synthstrip \'algorithm')
+  options.add_argument('-s', action='store_true', default=False, help='The output stripped image')
+  options.add_argument('-h', action='store_true',default=False, help='The help messeage of Synthstrip algorithm')
+  options.add_argument('-g', action='store_true',default=False, help='Use the GPU')
+  options.add_argument('-mo',action='store_true',default=False,help='Alternative model weights')
+  options.add_argument('-nocsf',action='store_true', default=False, help='Compute the immediate boundary of brain matter excluding surrounding CSF')
+  options.add_argument('-b', type=int,help='Control the boundary distance from the brain')
+  parser.add_argument('-clean_scale',
+                      type=int,
+                      default=DEFAULT_CLEAN_SCALE,
+                      help='the maximum scale used to cut bridges. A certain maximum scale cuts '
+                           'bridges up to a width (in voxels) of 2x the provided scale. Setting '
+                           'this to 0 disables the mask cleaning step. (Default: ' + str(DEFAULT_CLEAN_SCALE) + ')')
+  
+
+
+def get_inputs(): #pylint: disable=unused-variable
+  pass
+
+
+
+def needs_mean_bzero(): #pylint: disable=unused-variable
+  return True
+
+
+
+def execute(): #pylint: disable=unused-variable
+  
+  run.command('dwiextract input.mif - -bzero | mrmath - mean mean_bzero.mif -axis 3')
+  
+  run.command('mrconvert mean_bzero.mif 3dbzero.mif -axes 0,1,2')
+  app.cleanup('mean_bzero.mif')
+  run.command('mrconvert 3dbzero.mif 3dbzero.nii')
+  app.cleanup('3dbzero.mif')
+  
+  
+  synthstrip_cmd = shutil.which("mri_synthstrip")
```

Comment:
The name of the command is stored in the `SYNTHSTRIP` constant; so that should be used here.